lib/models/command.js

```diff
@@ -178,6 +178,38 @@ var Command = CoreObject.extend({
   },
 
   /**
+   * Called when command is interrupted from outside, e.g. ctrl+C or process kill
+   * Can be used to cleanup artifacts produced by command and control process exit code
+   *
+   * @method onInterrupt
+   * @return {Promise|undefined} if rejected promise then result of promise will be used as an exit code
+   */
+  onInterrupt: function() {
+    if (this._currentTask) {
+      return this._currentTask.onInterrupt();
+    }
```

Comment:
if we interrupt a command when there is no current task, should we expect a zero or nonzero exit code? If a user exists during init before task is started, it seems odd to me that we'd have a zero exit code in that scenario